pysat/instruments/cosmic_gps.py

```diff
@@ -29,12 +31,15 @@
 - Routine was not produced by COSMIC team
 
 """
+
 from __future__ import print_function
 from __future__ import absolute_import
 import glob
+import netCDF4
 import os
-import pandas as pds
+
 import numpy as np
+
```

Comment:
I believe numpy is a core import but netCDF4 is not, so their block locations should be changed.

Reply:
fixed